src/containers/HomePageContainer.tsx

```
@@ -0,0 +1,87 @@
+
+const ProductName = styled.h3`
+  font-size: ${theme.fontSizes.small}px;
+`
+
+const HomePageContainer = ({ products }: HomePageContainerProps) => {
+  const {clothesProducts, bookProducts, shoesProducts} = products;
+
+  const renderProductCardCarousel = (products: Product[]) => {
+    return (
+      <ProductList>
+        {products.map((p: Product, i: number) => (
```

ESLint

[error] 'i' is defined but never used.

(@typescript-eslint/no-unused-vars)